scripts/update/01-run.sh

```diff
@@ -54,6 +54,14 @@ if [[ ! -z "${UMBREL_OS:-}" ]]; then
         echo "ERROR: No Umbrel installation found at SD root ${SD_CARD_UMBREL_ROOT}"
         echo "Skipping updating on SD Card..."
     fi
+    
+    # Configure unattended-upgrades
+    # This can be safely removed from Umbrel v0.3.x+
+    echo "Configuring automated updates of apt packages"
+    apt-get update
+    DEBIAN_FRONTEND=noninteractive apt-get install unattended-upgrades -y
```

Comment:
It would be nice to wrap this in a check to see if the package is already installed to suppress extra unneeded log output.

We should also add this dep to Umbrel OS.